dev/knn.py

```
@@ -0,0 +1,197 @@
+        "reqCoordType": "WGS84GEO",
+        "searchOption": "0",
+        "resCoordType": "WGS84GEO",
+        "sort": "index"
+    }
+
+    headers = {
+        "accept": "application/json",
+        "content-type": "application/json",
+        "appKey": APP_KEY
+    }
+    response = requests.post(url, json=payload, headers=headers).json()
```

Ruff (0.15.2)

[error] Probable use of `requests` call without timeout

(S113)